src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ApplicationDocumentsTest.kt

```diff
@@ -39,16 +39,6 @@ class ApplicationDocumentsTest : InitialiseDatabasePerClassTestBase() {
         }
 
         val groupedDocuments = GroupedDocumentsFactory()
-          .withOffenderLevelDocument(
```

Comment:
can you recall why this was removed? given we're testing onlyConvictionDocuments = false i'd expect we'd want a non-conviction document in the results?